x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1391,4 +1391,202 @@ describe('when on the endpoint list page', () => {
       expect(startButton).toBeNull();
     });
   });
+  describe('endpoint list take action with RBAC controls', () => {
```

Comment:
Suggestion: most of the tests inside this `describe` are similar, could be interesting having a `describe.each` loop for those similars

Reply:
this is very true. i may try that in a separate pr!